### PR TITLE
ci: fixed android e2e CI. Emulator failing to boot.

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

See title.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [ ] I read and followed the [Flutter Style Guide].
- [ ] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [X] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/firebase/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: .github/workflows/e2e_tests.yaml [hunk 1]
```diff
@@ -55,14 +55,10 @@ jobs:
...
         with:
-          api-level: 30
-          target: google_atd
-          arch: x86
-          profile: Galaxy Nexus
-          force-avd-creation: false
-          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          disable-animations: false
-          channel: canary
+          api-level: 28
```
Comment (Lyokone) [inline .github/workflows/e2e_tests.yaml:58]: No way to go to api-level 33?

File: .github/workflows/e2e_tests.yaml [hunk 2]
```diff
@@ -55,14 +55,10 @@ jobs:
...
-          target: google_atd
-          arch: x86
-          profile: Galaxy Nexus
-          force-avd-creation: false
-          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
-          disable-animations: false
-          channel: canary
+          api-level: 33
+          arch: x86_64
+          target: google_apis
```
Comment (Lyokone) [inline .github/workflows/e2e_tests.yaml:60]: Looking at documentation `google_atd` was supposed to make tests go faster.
Do you think we should try `google_atd` with api-level 33?

https://github.com/marketplace/actions/android-emulator-runner#configurations

Comment (Lyokone) [reply .github/workflows/e2e_tests.yaml:60]: If it works with google_apis and is more stable it might not be worth it tho

Comment (Lyokone) [reply .github/workflows/e2e_tests.yaml]: @russellwheatley I think you need to add canary:
```
Note that aosp_atd and google_atd currently require the following: api-level: 30, arch: x86 or arch: arm64-v8 and channel: canary.
```

Comment (russellwheatley) [reply .github/workflows/e2e_tests.yaml:60]: Emulator fails to start with the `google_atd` setup, reverted back to what it was previously.

**Review by Lyokone — APPROVED** [review]

**Review by cabljac — APPROVED** [review]
lgtm

**Pull request merged**